assets/js/books.js

```diff
@@ -1,23 +1,64 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  title() {
+    return this.title;
+  }
+
+  author() {
+    return this.author;
+  }
+}
+
 const propBook = {
   submitButton: document.getElementById('button'),
   contentDiv: document.getElementById('content'),
   removeButtons: document.getElementsByClassName('remove'),
-  bookDivs: document.getElementsByClassName('books'),
   title: localStorage.getItem('title'),
   author: localStorage.getItem('author'),
 };
 
 const createBook = {
```

Comment:
- [ ] CreateBook function has to be inside a class as a class method but for now is outside kindly 
address that
- [ ] RemoveBook function is supposed to be inside a class as well as a class method
please work on that